tools/ci_build/github/azure-pipelines/publish-nuget.yml

```diff
@@ -2,10 +2,11 @@ resources:
   pipelines:
   - pipeline: build
     source: 'Zip-Nuget-Java-Nodejs Packaging Pipeline'
-    trigger: 
+    trigger:
```

Comment:
Skip the "report binary sizes" step, because we only gather data from the main branch. 
Also, please be aware that this pipeline could fail if a package with the same name is already published, which could happen if the package names do not contain a date string.